hyp3_rtc_gamma/__main__.py

```diff
@@ -96,7 +98,14 @@ def get_download_url(granule):
 def download_file(url):
```

Comment:
should probably move towards making this more library-ish:
`def download_file(url, retries=3, backoff=10, chunk_size=5242880):`